docs/workflows/phylogenetic_construction/snippy_variants.md

```diff
@@ -60,21 +60,21 @@ The `Snippy_Variants` workflow aligns single-end or paired-end reads (in FASTQ f
 
 ### Outputs
 
-!!! tip "Visualize your outputs in IGB"
+!!! tip "Visualize your outputs in IGV"
     Output bam/bai files may be visualized using IGV to manually assess read placement and SNP support.
```

Comment:
Can you add an admonition/call-out here specifying that the outputs for samtools coverage may not equal the percent reference coverage due to different calculation methods (breadth vs coverage)?